tools/filegone.py

```diff
@@ -47,11 +48,13 @@
 };
 
 BPF_PERF_OUTPUT(events);
+BPF_HASH(submit_data, u64, struct data_t);
```

Comment:
'submit_data' like a action. Rename it to 'currdata' (to represent the current 'data' indexed with tid)?

Reply:
Fix it, thank you, Song.